cli/jobs/pipelines/automl/image-classification-multiclass-pipeline.yml

```diff
@@ -0,0 +1,56 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+
+description: Image multiclass classification using pipeline job
+
+experiment_name: pipeline_image_multiclass_classification
+
+settings:
+  continue_on_step_failure: false
+  default_compute: azureml:gpu-cluster
+
+inputs:
+  image_multiclass_classification_train_data:
+    type: mltable
+    path: ../../automl-standalone-jobs/cli-automl-image-classification-multiclass-task-fridge-items/data/training-mltable-folder
+  image_multiclass_classification_validate_data:
+    type: mltable
+    path: ../../automl-standalone-jobs/cli-automl-image-classification-multiclass-task-fridge-items/data/validation-mltable-folder
+
+
+jobs:
+  automl_image_multiclass_classification_node:
+    type: automl
+    task: image_classification
+    log_verbosity: info
+    primary_metric: accuracy
+
+    limits:
+      timeout_minutes: 60
+
+    target_column_name: label
+    training_data: ${{parent.inputs.image_multiclass_classification_train_data}}
+    validation_data: ${{parent.inputs.image_multiclass_classification_validate_data}}
+
+    sweep:
+      limits:
+        max_trials: 20
+        max_concurrent_trials: 4
+      sampling_algorithm: Random
+      early_termination:
+        type: bandit
+        evaluation_interval: 10
+        slack_factor: 0.2
+
+    image_model:
+      checkpoint_frequency: 1
+      early_stopping: true
+      early_stopping_delay: 2
+      early_stopping_patience: 2
+      evaluation_frequency: 1
+
+    search_space:
```

Comment:
Would it be better to add the same search space as that in standalone jobs, for consistency ?